neurogym/wrappers/monitor.py

```diff
@@ -87,7 +130,7 @@ def step(self, action):
             self.store_data(obs, action, rew, info)
         if self.sv_stp == "timestep":
             self.t += 1
-        if info["new_trial"]:
+        if info.get("new_trial", False):
```

Comment:
this looks like it's the opposite of what it used to be. Or am I misinterpreting something?

Reply:
This safely checks (now) for the presence of the "new_trial" key. If it exists, it returns the value (True or False). If it doesn't, it returns the default value False. It enters in the if statement only if the key is present and its value is True (same behavior as before)